### PR TITLE
wow changes

### PR DESCRIPTION
**Pull request by RickEyre**

### CODE REVIEW

File: packages/panda-san/src/elements/button.scss [hunk 1]
```diff
@@ -5,9 +5,11 @@
   border: solid px-to-rem(2px) map-get($button-default-type, background-color);
   border-radius: px-to-rem(5px);
   color: map-get($button-default-type, color);
+  cursor: pointer;
   display: flex;
   justify-content: center;
   padding: px-to-rem(12px) px-to-rem(60px);
+  user-select: none;
```
Comment (ayeo127) [inline packages/panda-san/src/elements/button.scss:12]: ouu

File: packages/tiger-san/src/form/input.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    'input--select': type === 'select',
+    'input--textarea': type === 'textarea'
+  })
+)
+
+const getInputComponent = ({ type }) => {
+  return ['select', 'textarea'].includes(type) ? type : 'input'
+}
+
+export const Input = React.forwardRef(({
```
Comment (ayeo127) [inline packages/tiger-san/src/form/input.js:16]: what about other props like `onClick`, `onChange`, `disabled` ?

File: packages/tiger-san/src/form/label.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+
+const getLabelClassName = ({ className, visible }) => (
```
Comment (ayeo127) [inline packages/tiger-san/src/form/label.js:5]: ah, I like

File: packages/tiger-san/src/form/input.js (hunk 2 above)
Comment (ayeo127) [reply packages/tiger-san/src/form/input.js:16]: one other question, I'm guessing you'll be adding components like `InputSelect` and `InputTextArea` that returns `Input` with specific props. To handle that hierarchy, `Input` would need `{...props}` to map html element specific props.
e.g. `select` element has an attribute `multiple` and needs to be mapped in`Input`
e.g. `textarea` element has an attribute `rows` and needs to be mapped in `Input`

Comment (RickEyre) [reply packages/tiger-san/src/form/input.js:16]: Good question ... are you suggesting that we talk the approach:

```js
import { InputSelect, InputText, InputTextArea } from '@lemonpeach/tiger-san'

<Form>
  <InputSelect ... />
  <InputText ... />
```

I know this is how we have usually done this ... but I was taking a different approach here -- there would only be one `Input` and you would pass a `type` attribute` to it. `Input` component would handle the rest.

Internally we could have `InputSelect` handle that `type`, but I was thinking of maybe changing that too ... we could have some kind of "mixin" behaviour.

Thoughts?

Comment (ayeo127) [reply packages/tiger-san/src/form/input.js:16]: oh I see, I'm cool the new approach as long as `Input` doesn't get too verbose for handing various `types`!

**Pull request merged**